lib/main_common.pm

```diff
@@ -1848,6 +1848,10 @@ sub load_rollback_tests {
     }
 }
 
+sub load_extra_tests_himmelblau {
```

Comment:
This doesn't need a own subroutine IMHO.

Also I don't see this being used in the main call anywhere? How is this ever called? :thinking: